pymc/tests/test_distributions_moments.py

```diff
@@ -611,4 +612,28 @@ def test_hyper_geometric_moment(N, k, n, size, expected):
 def test_discrete_uniform_moment(lower, upper, size, expected):
     with Model() as model:
         DiscreteUniform("x", lower=lower, upper=upper, size=size)
+
+@pytest.mark.parametrize(
+    "a, size, expected",
+    [
+        (
+            np.array([2, 3, 5, 7, 11]), 
+            None, 
+            np.array([2, 3, 5, 7, 11])/28,
+        ),
+        (
+            np.array([[1, 2, 3], [5, 6, 7]]),
+            None,
+            np.array([[1, 2, 3], [5, 6, 7]])/np.array([6, 18])[..., np.newaxis],
+        ),
+        (
+            np.full(shape=np.array([7, 3]), fill_value=np.array([13, 17, 19])),
+            (11, 5,),
+            np.broadcast_to([13, 17, 19], shape=[11, 5, 7, 3]),
+        ),
```

Comment:
Hi @ricardoV94 , the third test here fails with error `ValueError: could not broadcast input array from shape (7,3) into shape (11,5)`. Is this expected?  If I put shape=`(11, 5, 7, 3)` instead of `(11, 5)`, my output shape is `(11, 5, 7, 3, 7, 3)` and I don't know how to write this test correctly

Reply:
I think that means you cannot simply use this lines to manage size information. That logic is only that straightforward for univariate distributions

```
if not rv_size_is_none(size):
    return at.full(size, moment)
```

Reply:
Got it, thanks for the pointer. I believe that I was able to fix this